### PR TITLE
GradientPicker: Auto-generate readme

### PR DESCRIPTION
**Pull request by mirka**
In preparation for #66974

## What?

Switch the README for GradientPicker to an auto-generated one (see also #66035).

## Testing Instructions

`npm run docs:components` to verify the changes to the README.

### CODE REVIEW

File: packages/components/src/gradient-picker/index.tsx [hunk 1]
```diff
@@ -166,44 +166,44 @@ function Component( props: PickerProps< any > ) {
 }
 
 /**
- *  GradientPicker is a React component that renders a color gradient picker to
+ * GradientPicker is a React component that renders a color gradient picker to
```
Comment (mirka) [inline packages/components/src/gradient-picker/index.tsx:169]: Fixed some formatting here.

File: packages/components/src/gradient-picker/index.tsx [hunk 2]
```diff
@@ -166,44 +166,44 @@ function Component( props: PickerProps< any > ) {
...
  * define a multi step gradient. There's either a _linear_ or a _radial_ type
  * available.
  *
  * ```jsx
- *import { GradientPicker } from '@wordpress/components';
- *import { useState } from '@wordpress/element';
+ * import { useState } from 'react';
```
Comment (mirka) [inline packages/components/src/gradient-picker/index.tsx:174]: We're switching our docs to import directly from `react`. (Looks like this was done in bulk for markdown files but not in JSDoc snippets?)

File: packages/components/src/gradient-picker/types.ts [hunk 3]
```diff
@@ -58,19 +58,17 @@ type GradientPickerBaseProps = {
 	loop?: boolean;
 } & (
 	| {
+			// TODO: [#54055] Either this or `aria-labelledby` should be required
```
Comment (mirka) [inline packages/components/src/gradient-picker/types.ts:61]: Moving the todo out of JSDoc.

**Review by tyxla — APPROVED** [review]
Looks great, thanks @mirka, 🚢 !

**Pull request merged**